wiki/Glossary/Antiflow/en.md

```diff
@@ -0,0 +1,9 @@
+---
+stub: true
+---
+
+# Antiflow
```

Comment:
I think this, along with `Snapping`, also covers osu! as a mode, so I would place that under `/wiki/Disambiguation/Antiflow` (I wish we also had `/wiki/Flow`!)